addons/web_editor/static/src/js/editor/snippets.editor.js

```diff
@@ -2869,9 +2869,9 @@ var SnippetsMenu = Widget.extend({
                     });
                     // If a modal is open, the scroll target must be that modal
                     const $openModal = self.getEditableArea().find('.modal:visible');
-                    if ($openModal.length) {
-                        self.draggableComponent.$scrollTarget = $openModal;
-                    }
+                    self.draggableComponent.$scrollTarget = $openModal.length
+                        ? $openModal
+                        : self.$scrollingElement;
```

Comment:
Hi @Zynton 
I revert a change you made in [this commit](https://github.com/odoo/odoo/commit/7ed3f3151a5352958f929ac8a4f10ea181be5ab1#diff-52a4f9d2c217548e69e6b7fd097f286f1754a6389734eea254b87255e501cbefL834)
I didn't understand why you changed it, I know it was a long time ago but do you have an explanation ?
If not, can you just tell me if you're ok with this PR?

Reply:
Hi @bvr-odoo
I think you're correct. It looks like an unfortunate attempt at a refactoring that assumed (probably from the comment) that `draggableComponent.$scrollTarget` had already been defined and that this line was just meant to redefine it in case a modal was open? I don't know, it's been a long time indeed. But seeing as `draggableComponent.$scrollTarget` doesn't seem to be defined anywhere else, reverting it seems right to me.
Just in case you didn't see btw, this exact change comes up twice in the same file.

Reply:
@Zynton Thanks for reply,
> Just in case you didn't see btw, this exact change comes up twice in the same file.

:+1:  Yes, I know the code is duplicated for "drag and drop" from snippet menu or "drag and drop" from the page.